services/processor/native/call.go

```diff
@@ -43,211 +48,54 @@ func processMethodCall(executionContextId primitives.ExecutionContextId, contrac
 	return contractOutputArgs, contractOutputErr, err
 }
 
-var bigIntType = reflect.TypeOf(big.NewInt(0))
-
-func prepareMethodInputArgsForCall(methodInstance types.MethodInstance, args *protocol.ArgumentArray, functionNameForErrors string) ([]reflect.Value, error) {
-	res := []reflect.Value{}
+func verifyMethodInputArgs(methodInstance types.MethodInstance, functionNameForErrors string, args []interface{}) ([]reflect.Value, error) {
```

Comment:
make sure that we have tests for:
- variadic with correct type
- variadic with incorrect type 
- variadic with both correct and incorrect types
- too many args
- not enough args

Reply:
all are present